src/phoenix/datasets/types.py

```diff
@@ -183,3 +208,202 @@ def __post_init__(self) -> None:
+                    "evaluator": run.name,
+                    "error": bool(run.error),
+                    "score": run.result.score if run.result else None,
+                    "label": run.result.label if run.result else None,
+                }
+                for run in eval_runs
+                if run is not None
+            ]
+        )
+        if df.empty:
+            df = pd.DataFrame.from_records(
+                [
+                    {"evaluator": name, "error": True, "score": None, "label": None}
+                    for name in config.eval_names
+                ]
+            )
+        stats = df.groupby("evaluator").agg(
+            n_errors=("error", "sum"),
+            **(
+                dict(
+                    n_scores=("score", "count"),
+                    avg_score=("score", "mean"),
+                )
+                if df["score"].any()
+                else {}
+            ),
+            **(
+                dict(
+                    n_labels=("label", "count"),
+                    top_2_labels=(
+                        "label",
+                        lambda x: dict(Counter(x).most_common(2)) if x.any() else None,
+                    ),
+                )
+                if df["label"].any()
+                else {}
+            ),
+        )  # type: ignore[call-overload]
+        sorted_eval_names = sorted(config.eval_names)
+        eval_names = pd.DataFrame(
+            {
+                "evaluator": sorted_eval_names,
+                "n": [config.exp_config.count] * len(sorted_eval_names),
+            }
+        ).set_index("evaluator")
+        stats = pd.concat([eval_names, stats], axis=1).reset_index()
+        summary: EvaluationSummary = object.__new__(cls)
+        summary.__init__(stats=stats)  # type: ignore[misc]
+        return summary
+
+    @classmethod
+    def __new__(cls, *args: Any, **kwargs: Any) -> Any:
+        # Direct instantiation by users is discouraged.
+        raise NotImplementedError
+
+    @classmethod
+    def __init_subclass__(cls, **kwargs: Any) -> None:
+        # Direct sub-classing by users is discouraged.
+        raise NotImplementedError
+
+
+@dataclass(frozen=True)
+class TaskSummary(_HasStats):
+    _title: str = "Experiment Summary"
+
+    @classmethod
+    def from_task_runs(
+        cls,
+        config: ExperimentConfig,
+        task_runs: Iterable[Optional[ExperimentRun]],
+    ) -> "TaskSummary":
+        df = pd.DataFrame.from_records(
+            [
+                {
+                    "example_id": run.dataset_example_id,
+                    "error": bool(run.error),
+                }
+                for run in task_runs
+                if run is not None
+            ]
+        )
+        n_runs, n_errors = len(df), 0 if df.empty else df["error"].sum()
+        record = {
+            "n": config.count,
+            "n_runs": n_runs,
+            "n_errors": n_errors,
+            "pct_errors": n_errors / n_runs,
```

Comment:
```suggestion
            "pct_error": n_errors / n_runs,
```

Reply:
removed it in favor of printing the top error string instead